packages/angular-playground/CHANGELOG.md

```diff
@@ -1,16 +1,29 @@
+# 5.1.1 (2018-08-16)
```

Comment:
Let's give schematics author (@sulco) kudos for his contribution to this release please.